css-pseudo-classes.tar/css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,23 @@
+th{
+  color:white;
+  background-color: black;
+}
+.team-1{
+  color:black;
+  background-color: rgb(217, 211, 211);
+}
+.team-2{
+  color:black;
+  background-color: white;
+}
```

Comment:
instead of using classes on the `td` elements... see if you can find a way to style the odd `td` rows by using a pseudo-class. See [https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child](url)

Hint: you can use `:nth-child(odd)` in your pseudo-class selector